packages/wrangler/src/containers/containers.ts

```diff
@@ -147,6 +156,147 @@ export async function listCommand(
 	await listCommandHandle(listArgs, config);
 }
 
+export function sshYargs(args: CommonYargsArgv) {
+	return (
+		args
+			.positional("ID", {
+				describe: "id of the container instance",
+				type: "string",
+				demandOption: true,
+			})
+			// Following are SSH flags that should be directly passed in
+			.option("c", {
+				describe: "cipher_spec",
```

Comment:
```suggestion
				describe: "ssh option for  cipher_spec",
```

and for the rest too